src/ZatData.cpp

```diff
@@ -591,6 +593,9 @@ ZatData::ZatData(const string& u, const string& p, bool favoritesOnly,
   }
   
   ReadDataJson();
+  if (!xmlTVFile.empty()) {
+    xmlTV = new XmlTV(xmlTVFile);
+  }
```

Comment:
Unlike in Java, in  c++ it is common practice to put  curly braces in the next line.